src/renderer/shims/LocalShim.ts

```diff
@@ -81,12 +81,24 @@ export class LocalShim implements SimulatorInterface {
     public Aircraft = {}
 
     public GetStoredData = (key: string): any => {
-        if (key === 'A32NX_CONFIG_SELF_TEST_TIME') {
-            return 2;
+        // if (key === 'A32NX_CONFIG_SELF_TEST_TIME') {
+        //    return 2;
+        //}
+
+        try {
+            const item = window.localStorage.getItem(key);
+            return item ? JSON.parse(item) : null;
+        } catch (error) {
+            console.log(error);
+            return 0;
         }
-        return 0;
     }
 
-    public SetStoredData = () => {
+    public SetStoredData = (key: string, value: any): void => {
+        try {
+            window.localStorage.setItem(key, JSON.stringify(value));
```

Comment:
Only stores strings... passing object or number results in the value not being set (function still returns null though).

Reply:
I do not get this one: in case a number or object is provided as value, JSON.stringify() returns a value? Maybe I miss somethig?
Reg. `null` return: Will change the method declaration do return `any` as defined in dataStorage.d.ts and will return `null`. Was overseeing this one.

Reply:
You can only store a string. If you pass any other type it will silently fail. Try it in-sim. ;)

Reply:
Then we should also just use strings and error out in case a non-string is provided as value. What do you think?

Reply:
Yeah, I guess it would be good to throw in ACE so you can notice such a bug. 😁